packages/kbn-alerts-ui-shared/src/rule_type_modal/components/rule_type_list.tsx

```diff
@@ -47,7 +47,12 @@ export const RuleTypeList: React.FC<RuleTypeListProps> = ({
   onClearFilters,
   showCategories = true,
 }) => {
-  const ruleTypesList = [...ruleTypes].sort((a, b) => a.name.localeCompare(b.name));
+  const ruleTypesList = [...ruleTypes].sort((a, b) => {
+    if (a.enabledInLicense === b.enabledInLicense) {
```

Comment:
super nit: What do you think of extracting the logic to a helper function (inside the component) or add a comment explaining the logic?

Reply:
+1 for the comment in the code

Reply:
done in e9812bf6a4b12db7df4bdc1bff4f0e34c21a2ded